README.md

```diff
@@ -28,6 +28,37 @@ Major components of the project include:
 - [oli](bin/oli): OpenDAL Command Line Interface
 - [oay](bin/oay): OpenDAL Gateway
 
+**Services**
+
```

Comment:
It would probably be better to list the general backend and the kv backend separately.

Reply:
Also this will be a relatively long list, I personally tend to list a few of the main ones and then fold the rest using `<details close><summary></summary></details>`.

Reply:
> It would probably be better to list the general backend and the kv backend separately.

Why? OpenDAL users shouldn't care about this.

Reply:
> Also this will be a relatively long list, I personally tend to list a few of the main ones and then fold the rest using `<details close><summary></summary></details>`.

Since it's bot long enough, let's list them all AS-IS.

Reply:
> Since it's bot long enough, let's list them all AS-IS. They are OpenDAL's strength.

Maybe we can have a better organizations, let me take a look.

Reply:
We may also add a brief introduction about services here.(Or a link to docs)

Reply:
> We may also add a brief introduction about services here.(Or a link to docs)

This is the README for OpenDAL project so we should not link to rust core's docs. We just let users know OpenDAL supports the following services.